src/hooks/project-hooks.tsx

```diff
@@ -85,6 +85,26 @@ interface ProjectProviderProps {
   children: ReactNode;
 }
 
+const useDefaultProjectName = (): string => {
+  const intl = useIntl();
+  return intl.formatMessage({ id: "default-project-name" });
+};
+
+export const useHasUntitledProjectName = (): boolean => {
```

Comment:
Feels a bit of a mouthful. Maybe useProjectIsUntitled ?

Reply:
[337f1d7](https://github.com/microbit-foundation/ml-trainer/pull/462/commits/337f1d783778d99331ac2f97a692170f490bbc44)